mandatory/1-syntax-errors.js

```diff
@@ -1,14 +1,14 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
-
+function introduceMe(name, age) {
+  return "Hello, my name is " + name + "and I am " + age + "years old";
```

Comment:
please try using DevTools console in your browser to test this function. 
```js
function introduceMe(name, age) {
  return "Hello, my name is " + name + "and I am " + age + "years old";

introduceMe("Your Name", youAge);
}
```
what output do you see?
is there any small typo in it?

Reply:
There's no spaces between strings